kolibri/core/templates/kolibri/base.html

```diff
@@ -18,10 +15,11 @@
 </ul>
 {% endblock %}
 
-<hr>
-
-{% block content %}
-
+<main></main>
+{% block frontend_assets %}
```

Comment:
Well, to be honest I expected the `ready` method to execute after the page has loaded. But it does not. Does it have a well-defined time when it's executed? Can we just make *that* do its thing after page load?